FreeRTOS/FreeRTOS-Kernel
# Add CC-RH port for Renesas F1Kx devices

Add CC-RH port for Renesas F1Kx devices

Description
-----------
Add port files (port.c, portasm.s, portmacro.h) for CC-RH core for Renesas F1Kx devices.

Test Steps
-----------
The test project can be found [here](https://github.com/FreeRTOS/FreeRTOS-Community-Supported-Demos) (`RH850_F1Kx_CCRH`). This project contains example tasks and configurations to help you get started with FreeRTOS on the RH850/F1K and F1Kx.

Checklist:
----------



Related Issue
-----------
None


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
